src/chain_sync/tipset_syncer.rs

```diff
@@ -103,6 +104,8 @@ pub enum TipsetRangeSyncerError {
     TipsetParentNotFound(ChainStoreError),
     #[error("Consensus error: {0}")]
     ConsensusError(FilecoinConsensusError),
+    #[error("Other error: {0}")]
+    Other(String),
```

Comment:
Which errors go here?

Reply:
It's `JoinError`. `JoinSet::poll_join_next` returns`Option<Result<Result<(), SyncRangeError>,JoinError>>`

Reply:
Let's make it explicit that it is a join error.

Reply:
Or better yet: Propagate the panic.

Reply:
Fixed